charts/postgres-db-sync/README.md

```diff
@@ -12,5 +12,5 @@ to extract the records from the source database/tables.
 - `createTargetTableConfigMap`: should point to a Configmap that contains a file named `create_target.sql` with the SQL code
```

Comment:
The readme needs to be updated: with the current code in this PR, you do not need this value, and the `customExportStatementConfigMap` must contain 3 things: 
 - export.sql
 - export_all.sql
 - create_target.sql which has some quite specific requirement